src/Generator/Passes/CleanCommentsPass.cs

```diff
@@ -54,6 +54,23 @@ public bool VisitInlineCommand(InlineCommandComment comment)
 
         public bool VisitParagraphCommand(ParagraphComment comment)
         {
+            for (int i = 0; i < comment.Content.Count; i++)
+            {
+                if (comment.Content[i].Kind == DocumentationCommentKind.InlineCommandComment)
+                {
+                    TextComment com = (TextComment) comment.Content[i + 1];
```

Comment:
I think this code is safe but still, as we discussed, it would be really pathetic if we crashed because of a comment. So add guards for the bounds and the type.